packages/spacetimedb.bsatn.runtime/1.0.0-rc4.meta

```diff
@@ -1,8 +0,0 @@
-fileFormatVersion: 2
```

Comment:
All these .meta removals need to be undone, I didn't understand what I was doing and broke unity.